app/views/users/_form.haml

```diff
@@ -18,8 +18,10 @@
       %td= f.label :hastus_id
       %td= f.text_field :hastus_id
     %tr
-      %td= f.label :division
-      %td= f.select :division, options_for_select(User::DIVISIONS)
+      %td= f.label :division_ids
+      %td= f.select :division_ids,
+        options_from_collection_for_select(Division.order(:name), :id, :name, user.divisions.pluck(:id)),
+        { }, multiple: true
```

Comment:
After `options_from_collection...`, there are *two* hashes. The first is internal options, the second is external (HTML) options. This is the only way (that I know of) to make sure that the `multiple: true` ends up in the external options hash.

Reply:
It's the only way that I know of as well.